frontend/packages/console-shared/src/types/resource.ts

```diff
@@ -1,4 +1,5 @@
 import { K8sResourceKind, PodKind, RouteKind } from '@console/internal/module/k8s';
+import { Pipeline, PipelineRun } from '@console/dev-console/src/utils/pipeline-augment';
```

Comment:
This is a bad dependency.
`console-shared` cannot depend on `dev-console`.
Look at how knative plugs into the overview.

Reply:
that can be achieved from `plugin.tsx` in dev-console as an extension, but going forward will it be a good idea to have tekton as seperate plugin llike kn.

Reply:
Yup I agree. I told the same thing to @sahil143. But for now pipelines being in dev-console is fine until we get time to refactor it out.